bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/types/EventDescription.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.types;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Describes event options and gives information how to interpret it.
+ */
+public class EventDescription {
+    private final List<StateOption> options;
```

Comment:
Although the class looks almost the same, I would suggest to introduce an EventOption here, since State is simply not correct.